api/Requests.ts

```diff
@@ -56,6 +56,12 @@ export const startTournament = async (
   return resp;
 }
 
+export const deleteTournament =async (tournamentId: string, user: User) => {
+  const url = `${apiHost}/tournaments`
+  const resp = await deleteRequest(url, {"id": tournamentId}, user)
```

Comment:
Generally actions on a specific tournament should have the `id` provided in the path, like the start tournament request above. So we'd probably want this to look more like:
 ```suggestion
  const url = `${apiHost}/tournaments/{tournamentId}`
  const resp = await deleteRequest(url, {}, user)
```